folly/CPortability.h

```diff
@@ -240,3 +240,11 @@
 #else
 #define FOLLY_ERASE_TRYCATCH FOLLY_ERASE
 #endif
+
+//  In the recent FreeBSD releases, malloc.h header does exist
+//  however when __STDC__ is defined it throws an exception
+#ifdef __FreeBSD__
+#ifdef __STDC__
+#undef __STDC__
```

Comment:
What defines `__STDC__`?

Reply:
I wanted to avoid forcing specific compiler flags.

Reply:
Overall in folly, we should stick to defining/undefining only names that are namespaced to folly.